policyengine_us/variables/gov/states/ct/tax/income/subtractions/ct_magi_excess_over_base.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class ct_magi_excess_over_base(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Excess of modified adjusted gross income and social security over base amount calculation for the Connecticut social security subtraction"
+    reference = (
+        # Connecticut General Statutes, Chapter 229, Sec. 12-701, (20), (b), (x), (iii) and (iv)
+        "https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-701"
+        # Section 86(b)(1) of the Internal Revenue Code
+        "https://www.law.cornell.edu/uscode/text/26/86"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    # Code extracted from tax_unit_taxable_social_security
+    def formula(tax_unit, period, parameters):
```

Comment:
@MaxGhenis we would need to refactor `tax_unit_taxable_social_security` to derive this without duplicating code. 
I would keep the current structure and file an issue to consolidate once this has been addressed as we will also need to consider the in the Montana SS structure 

[Issue](https://github.com/PolicyEngine/policyengine-us/issues/3337) filed

Reply:
I think refactoring here would help clarify. The code+comments don't make it clear how this program works, which led to at least one significant error. Please clarify in some way and let me know when it's ready for another.